src/test/java/database/dataprovider/EmployeeSQLDataProvider.java

```diff
@@ -0,0 +1,75 @@
+package database.dataprovider;
+
+import database.DataBaseManager;
+import database.interfaces.VacancyDAO;
+import selenium.common.Utils;
+import selenium.model.Candidate;
+import selenium.model.Vacancy;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmployeeSQLDataProvider extends DataBaseManager implements VacancyDAO {
+    private List<Candidate> candidateList = new ArrayList<>();
+    private Candidate candidate;
```

Comment:
this is redundant here cause u return all data form methods

Reply:
fixed